ui/src/common/ArtistShow.js

```diff
@@ -0,0 +1,388 @@
+    },
+    cover: {
+      width: 151,
+      boxShadow: '1px 1px 20px 0px #565656',
+      borderRadius: '5px',
+    },
+    artImage: {
+      marginTop: '1rem',
+      marginLeft: '1em',
+      maxHeight: '10rem',
+      backgroundColor: 'inherit',
+      display: 'flex',
+      [theme.breakpoints.down('xs')]: {
+        marginTop: '4rem',
+        maxHeight: '7rem',
+        width: '7rem',
+      },
+    },
+    artDetail: {
+      margin: '1rem',
+      flex: '1',
+      display: 'flex',
+      minHeight: '10rem',
+      [theme.breakpoints.down('xs')]: {
+        display: 'none',
+      },
+    },
+    expand: {
+      display: 'flex',
+      padding: '0',
+      boxShadow: 'none',
+      backgroundColor: 'inherit',
+      fontSize: '0.77rem',
+      color: '#a0a0a0',
+      border: 'none',
+      '& .MuiButton-label': {
+        display: 'contents',
+      },
+      '&.MuiButton-contained:hover': {
+        boxShadow: 'none',
+        backgroundColor: 'inherit !important',
+        color: '#dbdada',
+      },
+    },
+    less: {
+      display: 'none',
+      [theme.breakpoints.down('xs')]: {
+        display: ({ link }) => (link ? 'flex' : 'none'),
+        width: '7rem',
+        flex: '1',
+        alignItems: 'flex-end',
+        padding: '0',
+        marginTop: 'auto',
+        border: 'none',
+        boxShadow: '-10px 0px 18px 5px black',
+        background: 'inherit',
+        textTransform: 'capitalize',
+        '&:hover': {
+          background: 'black',
+          boxShadow: '-10px 0px 18px 5px black',
+        },
+        '& .MuiButton-label': {
+          color: `${theme.palette.primary.main}!important`,
+        },
+      },
+    },
+    more: {
+      display: 'none',
+      [theme.breakpoints.down('xs')]: {
+        display: ({ link }) => (link ? 'flex' : 'none'),
+        width: '7rem',
+        flex: '1',
+        alignItems: 'flex-start',
+        padding: '0',
+        marginBottom: 'auto',
+        border: 'none',
+        boxShadow: '-10px 0px 18px 5px black',
+        background: 'inherit',
+        textTransform: 'capitalize',
+        '&:hover': {
+          background: 'black',
+          boxShadow: '-10px 0px 18px 5px black',
+        },
+        '& .MuiButton-label': {
+          color: `${theme.palette.primary.main}`,
+        },
+      },
+    },
+    album: {
+      marginBottom: '1em',
+    },
+  }),
+  { name: 'NDArtistPage' }
+)
+
+function ImgMediaCard({ artId, artist }) {
+  const [lastInfo, setlastInfo] = useState()
+  const [expanded, setExpanded] = useState(false)
+
+  const props = { ...artist }
+  const artistProps = props['0']
+  var title = artistProps?.artist
+  var lastLink = ''
+  const link = lastInfo?.biography?.match(
+    /<a\s+(?:[^>]*?\s+)?href=(["'])(.*?)\1/
+  )
+  const biography = lastInfo?.biography?.replace(new RegExp('<.*>', 'g'), '')
+
+  const handleExpandClick = useCallback(() => {
+    setExpanded(!expanded)
+  }, [expanded, setExpanded])
+
+  try {
```

Comment:
The try...catch should be inside the `useEffect` block.

Actually, instead of try...catch, you can simply add a `.catch(e=>{})` block after the last `.then` to capture any exceptions

Reply:
Ok, I'll try removing the try...catch block, and let you know the results.